packages/telemetry/src/ingest-slog-entrypoint.js

```diff
@@ -29,6 +30,10 @@ async function run() {
     return;
   }
 
+  if (args[0] === '-') {
+    args[0] = '';
+  }
```

Comment:
I'm hesitant to mutate the args array, because it implies that there is non-local magic that requires the mutation. 
  After investigating, I found no other use of `args[0]` than just to initialize `slogFile`.  What about:
```suggestion
  const slogFile = args[0] === '-' ? '' : args[0];
```

Reply:
The args are already local, but yeah, I can do that.